Libs/MRML/Core/vtkMRMLClipModelsNode.cxx

```diff
@@ -94,6 +96,12 @@ void vtkMRMLClipModelsNode::ReadXMLAttributes(const char** atts)
       ss << attValue;
       ss >> ClipType;
       }
+    else if (!strcmp(attName, "clippingMethod"))
+      {
+      std::stringstream ss;
+      ss << attValue;
+      ss >> ClippingMethod;
```

Comment:
Will have to read string and assume associated enum value if above comment changes are made.